File: chegaja_frontend/lib/models/entregador.dart
class Entregador {
  String? email;
  String? senha;
  String? telefone;
  String? nome;
  String? cpf;
  int? pesoMaximo;

  Entregador(
      {this.email,
      this.senha,
      this.telefone,
      this.nome,
      this.cpf,
      this.pesoMaximo});

  Entregador.fromJson(Map<String, dynamic> json) {
    email = json['email'];
    senha = json['senha'];
    telefone = json['telefone'];
    nome = json['nome'];
    cpf = json['cpf'];
    pesoMaximo = json['pesoMaximo'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['email'] = this.email;
    data['senha'] = this.senha;
    data['telefone'] = this.telefone;
    data['nome'] = this.nome;
    data['cpf'] = this.cpf;
    data['pesoMaximo'] = this.pesoMaximo;
    return data;
  }
}

File: chegaJa_back/src/main/java/com/tech/chegaJa/domain/model/Entregador.java
package com.tech.chegaJa.domain.model;

import com.tech.chegaJa.domain.dto.EmpresaDto;
import com.tech.chegaJa.domain.dto.EntregadorDto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.math.BigDecimal;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Entregador {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String nome;
    private String cpf;
    private BigDecimal pesoMaximo;
    private BigDecimal latitude;
    private BigDecimal longitude;

    public void setCnpj(String cpf) {

    }

    public EntregadorDto toDto(){
        return new EntregadorDto(this.id,this.nome,this.cpf, pesoMaximo);
    }
}
